Source/Scene/Fog.js

```diff
@@ -131,7 +131,7 @@ define([
 
         // Fade fog in as the camera tilts toward the horizon.
         var positionNormal = Cartesian3.normalize(camera.positionWC, scratchPositionNormal);
-        var dot = CesiumMath.clamp(Cartesian3.dot(camera.directionWC, positionNormal), 0.0, 1.0);
+        var dot = Math.abs(Cartesian3.dot(camera.directionWC, positionNormal));
```

Comment:
This change was from this comment: https://github.com/AnalyticalGraphicsInc/cesium/pull/4307#r78224791. The previous math resulted in fog being fully applied unless looking straight at the horizon or at the sky.